### ISSUE TITLE
Version preference prefers RC version over GA

### ISSUE THREAD

**Issue by vsalbaba** [wildfly-extras/wildfly-channel#50]

**Comment by jmesnil** [comment]
That's correct. That's why we use "CR1" for candidate release and not "RC1"

**Comment by vsalbaba** [comment]
Another example from `wildfly-ee-galleon-pack-8.0.0.Beta-redhat-20220526-channel.yaml`

```
  - groupId: org.apache.taglibs
    artifactId: taglibs-standard-compat
    version: "1.2.6.RC1-redhat-1"
```

**Comment by vsalbaba** [comment]
This issue is closed because production channels must always specify exact version making the issue irrelevant. In development where versionPattern might be used, the issue is rare and acceptable.
